DuckDuckGo/UserText.swift

```diff
@@ -1358,4 +1358,43 @@ But if you *do* want a peek under the hood, you can find more information about
             static let daxDialogBrowsingWithMultipleTrackers = NSLocalizedString("contextual.onboarding.browsing.multiple.trackers", comment: "First parameter is a count of additional trackers, second and third are names of the tracker networks (strings)")
         }
     }
+
+    public enum HighlightsOnboardingExperiment {
+        enum Intro {
+            public static let title = NSLocalizedString("onboarding.highlights.intro.title", value: "Hi there.\n\nReady for a faster browser that keeps you protected?", comment: "The title of the onboarding dialog popup")
+        }
+        
+        enum BrowsersComparison {
+            enum Features {
+                public static let cookiePopups = NSLocalizedString("onboarding.highlights.browsers.features.cookiePopups.title", value: "Block cookie requests & popups", comment: "Message to highlight browser capability of blocking cookie pop-ups")
```

Comment:
Maybe you could phrase the comment to say Message to highlight how the browser allows you to block… cause I think he intention is to be like:
Blocca gli ads! etc 

Just a thought in cases helps with translations